.travis.yml

```diff
@@ -42,6 +42,9 @@ install:
   # Ensure postgres user has permissions
   - docker exec -it pgbuild /bin/bash -c "chown -R postgres:postgres /build/"
   - docker exec -it pgbuild /bin/bash -c "chown -R postgres:postgres /postgres/"
+before_script:
+  - sudo mkdir -p -m 0777 /cores
+  - sudo echo '/cores/core.%e.%p' | sudo tee /proc/sys/kernel/core_pattern
```

Comment:
why not `sudo echo '/cores/core.%e.%p' > /proc/sys/kernel/core_pattern`

Reply:
Unfortunately otherwise it prints some permission errors

Reply:
That is because the redirection (`>`) does not run under `sudo` privileges, but `tee` does.

Reply:
but then you can skip the initial sudo since you dont need permissions for the echo

Reply:
But you need permissions to open `/proc/sys/kernel/core_pattern`, right?